### ISSUE TITLE
Plotting / bar charts?

### ISSUE THREAD

**Issue by aguformoso** [thybag/ha-energy-entity-card#3]
Thanks for your contribution! Very useful. It would be a great addition if besides calculating the entity value for the selected time range, the entity would be plotted (e.g. bar charts in the Home Assistant energy dashboard).

**Comment by Fl0x23** [comment]
I have to agree with the previous speaker. It would be very helpful if the entity value for the selected time range and the entity could also be displayed graphically, as in the original energy dashboard, or is there already another solution for this?

**Comment by thybag** [comment]
I've had a quick look at whether or not it would be possible for this card to pass date range information down in to one of the existing chart cards, but of the two i looked at https://github.com/kalkih/mini-graph-card & https://github.com/RomRider/apexcharts-card, it doesn't appear that either of them allow for a configured start or end date.

If your aware of any existing charts that do allow for a start/end or similar values to be configured, I'd be open to seeing if I can make this card passively pass that down. 

As for adding dedicated charting capabilities, I think it'd likely make more sense for the functionality of my card to be added to one of the existing chart cards about (given these already have a rich set of charting capabilities built).

**Comment by scutzi128** [comment]
You can use the statistics graph card and add "energy_date_selection: true " to allow the graph to also follow the energy date selector card.

Like so:

![Image](https://github.com/user-attachments/assets/4fcd79dd-bba0-4633-a35b-eb91d5708b50)

**Comment by aguformoso** [comment]
Thanks @scutzi128! Exactly what I was needing. After updating my HA instance `energy_date_selection` started working.
